tests/test_cases/test_array/test_array.py

```diff
@@ -326,22 +326,22 @@ def test_discover_all(dut):
     elif cocotb.LANGUAGE in ["verilog"] and cocotb.SIM_NAME.lower().startswith(("riviera")):
         if cocotb.SIM_VERSION.startswith(("2017.10.61")):
             pass_total = 803
-        elif cocotb.SIM_VERSION.startswith(("2016.06","2016.10","2017.02")):
+        elif cocotb.SIM_VERSION.startswith(("2016.06", "2016.10", "2017.02")):
```

Comment:
This is good whitespace cleanup, but this PR is not the place for it. Can you move these trivial style changes to a separate PR, so that it makes the non-trivial content of this one easier to review?

Reply:
Sure thing, I just did some of them so my linter would stop complaining at me!

Reply:
All the style changes, or just ones in standalone files?

Reply:
https://github.com/potentialventures/cocotb/pull/1022

Once that's merged to master, I'll merge it here and it should remove a lot of the whitespace changes.